tests/fixtures/client.py

```diff
@@ -106,13 +108,18 @@ async def func(
             enable_api=False,
             groups=None,
             permissions=None,
-            use_b2c=False
+            use_b2c=False,
+            base_url="https://app.virtool.ca",
+            add_route_table: RouteTableDef = None
```

Comment:
Should be `addon_route_table`. Add is a verb which makes this sound like it should be a function.